lib/stack_master/commands/apply.rb

```diff
@@ -12,7 +12,7 @@ def initialize(config, stack_definition, options = Commander::Command::Options.n
         @stack_definition = stack_definition
         @from_time = Time.now
         @options = options
-        @options.on_failure ||= "ROLLBACK"
+        @options.on_failure ||= @config.stack_defaults['on_failure']
```

Comment:
@patrobinson Ah right, but you mean `@option.on_failure ||= nil` here?
If so, since currently `use_change_set?` returns `@option.on_failure.nil?`, do users always have to provide `--on-failure` option to call `create_stack_directly` for non-SAM templates when running `apply` command?